src/otx/algo/detection/atss.py

```diff
@@ -183,7 +183,6 @@ def _exporter(self) -> OTXModelExporter:
                 "input_names": ["image"],
                 "output_names": ["boxes", "labels"],
                 "dynamic_axes": {
-                    "image": {0: "batch", 2: "height", 3: "width"},
```

Comment:
I looks a bit weird that the batch dim is static on input, but dynamic on output. What is the actual outcome in terms of i/o shapes?

Reply:
OV does support dynamic input shapes; however, we have performance issues I think, which is why we currently fix the input shape. This could be made configurable in the future, there is no immediate use case afaik. For now, the static input shape is determined based on the input tensor during tracing. Regarding the output, it remains dynamic, upto the number of max pred.

Reply:
https://github.com/openvinotoolkit/training_extensions/issues/4111 was related to OV preprocessor insertion failure: it could not deduct h w required to initialize the resize.
I think dynamic batch is ok here, since OV can set it to 1 on it's side.
Now, logically, the export config doesn't make sense to me: it says the model has fixed input batch dim, but dynamic output batch. OV eats that, because it does shape inference on it's side, but overall the configuration looks inconsistent: the batch axis should be either kept everywhere, or deleted from the config.